ultralytics/engine/exporter.py

```diff
@@ -72,6 +72,8 @@
 from ultralytics.utils import (
     ARM64,
     DEFAULT_CFG,
+    IS_JETSON,
```

Comment:
I think I need to revert all changes to exporter.py that you've made here, because they are all just swapping ARM64 for RPI and Jetson constants, but this doesn't make any sense to me.

Reply:
okay please go ahead. Let's stick with ARM64 here then. My bad to make things too complicated. @glenn-jocher